skater/player.py

```diff
@@ -22,27 +22,18 @@ def __init__(self, speed = 0):
 
         self.image = pygame.image.load(images.PLAYER_MAIN).convert_alpha()
         self.rect = self.image.get_rect()
-        
-        # make collider smaller than the image
-        band = 4
-        # self.collide_rect = pygame.Rect(self.rect.x + band, self.rect.top, self.rect.width - 2*band, self.rect.height)
 
         # speed is used for movement to left/right
         self.speed = speed
 
         # acceleration, velocity, mass - used for acceleration and deceleration. 
         # separate velocities for movement on x axis (right/left) and y axis (jump)
-        self.a = 3
         self.v_x = self.v0 = speed * 1.8
         self.v_y = self.G * self.v0
         self.m = 4
-        self.is_jumping = False
-        self.is_falling = False
-        self.is_decelerating = False
 
         # flags used to perform tricks
         self.is_manual = False
```

Comment:
to chiba tyz bedzie trzeba przepisac na metode

Reply:
Tak, w zasadzie wszystkie is_* lepiej żeby były metodami.
Jedynie is_manual / cokolwiek w jakiś sposób będziemy musieli przechowywać w stanie, ale to raczej nie jako `is_*` tylko dodamy jeden parametr wskazujący obecnie wykonywaną sztuczkę / co tam można robić.